test/utils/unit-test-helpers.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2019 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* istanbul ignore file */
+
+const missingParamsError = 1;
+const missingParamsSuccess = 0;
+
+module.exports.missingParamsError = missingParamsError;
+module.exports.missingParamsSuccess = missingParamsSuccess;
+
+module.exports.checkUrlAndMethod = function(options, url, method) {
```

Comment:
I think we should have a description on any method that is exported from the package.  This doesn't need to be elaborate -- maybe just a one sentence statement of behavior.

Reply:
Agreed, I will add that